code/modules/crafting/craft.dm

```diff
@@ -54,7 +54,9 @@
 /datum/personal_crafting/proc/check_contents(datum/crafting_recipe/R, list/contents)
 	contents = contents["other"]
 	main_loop:
-		for(var/A in R.reqs)
+		for(var/atom/A in R.reqs)
+			if (A.isInCraftingBlacklist(R))
```

Comment:
Uh, shouldn't this be checking the actual item, not requirement ?

Reply:
hm
perhaps i am braindead

Reply:
this is gonna need a bit of work

Reply:
@ExcessiveUseOfCobblestone this code is much harder to deal with than that, im gonna need to do some different shit

Reply:
I'll give it the cobblestone pass (tm) this evening.

Reply:
i was playing heroes of the storm whoops